Makefile

```diff
@@ -149,8 +149,23 @@ release-src:
 .PHONY: gen-tools
 gen-tools:
 	go mod download
-	go install k8s.io/code-generator/cmd/{client-gen,lister-gen,informer-gen,deepcopy-gen}
+	@bash -c 'go install k8s.io/code-generator/cmd/{client-gen,lister-gen,informer-gen,deepcopy-gen}'
 
 .PHONY: codegen
 codegen: gen-tools
 	./utils/update-codegen.sh
+
+.PHONY: verify-codegen
+verify-codegen: gen-tools
```

Comment:
Add comments for these new directives so that their functions can be shown when people executing `make help`.

Example:

```
### verify-codegen:          Verify whether the generated codes are valid.
verify-codegen: gen-tools
```